presto-main/src/main/java/com/facebook/presto/operator/scalar/TryCastFunction.java

```diff
@@ -45,7 +45,7 @@
     public TryCastFunction()
     {
         super(new Signature(
-                "TRY_CAST",
+                TRY_CAST_NAME,
```

Comment:
NVM, you fixed in a later commit.

Reply:
Didn't realize it's the write commit. I'll fix that.